operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -1,18 +1,14 @@
 # Documentation Style Guide
 
-When writing for Mattermost, always think about the audience that will be reading your writing. Mattermost’s 
-audience can vary, but we typically communicate with a technical audience and want to present Mattermost as 
-being a friend of the practitioner in development, IT, and operations. It must be accurate and clear, be 
-presented with a style and tone that is appropriate for technical content, and easy to navigate.
+When writing for Mattermost, always think about the audience that will be reading your writing. Mattermost’s audience can vary, but we typically communicate with a technical audience and want to present Mattermost as being a friend of the practitioner in development, IT, and operations. It must be accurate and clear, be presented with a style and tone that is appropriate for technical or educational content, and easy to navigate.
+
+When writing educational content the audience is placed at the center of attention, opposing technical documentation where it is the documents content.
```

Comment:
```suggestion
An exception is Customer Education content where the audience is placed at the center of attention, as opposed to technical documentation where it is the document's content that is the primary focus.
```
Do you have examples of both? As they are two different ideas it might be helpful to a new writer or contributor to have a point of reference.

Reply:
How about 

> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration. 
>
>We are here to [support](https://link-to-support-section) you.

As opposed to:
> Deploying the Mattermost Mobile App will take time and iteration.

We could include or refer to it in the Voice section, WDYT @justinegeffen

Reply:
Would the above examples be related to this paragraph or should they be added to the voice section? They would fit better in the voice section if they're not already there.

Reply:
this example is part of this pr in the voice section